infrastructure/alerts.tf

```diff
@@ -48,3 +48,39 @@ module "fpl-grant-case-access-failure-alert" {
   resourcegroup_name         = "${local.alert_resource_group_name}"
   enabled                    = "${var.enable_alerts}"
 }
+
+module "fpl-exceptions-alert" {
+  source                     = "git@github.com:hmcts/cnp-module-metric-alert"
+  location                   = "${var.appinsights_location}"
+  app_insights_name          = "${var.product}-${var.component}-appinsights-${var.env}"
+  alert_name                 = "${var.product}-exceptions-alert"
+  alert_desc                 = "All exceptions within FPL"
+  app_insights_query         = "exceptions | where operation_Name !contains "health" | project timestamp, severityLevel, operation_Name, outerMessage, ['details'] | sort by severityLevel"
```

Comment:
How severrity level is controlled? I thought it is simply loggin all the errors so not sure what information it provides can you provide some info? 

I see details might be too much - it provides the whole stack which we don't need here. 
Please add operation_Id I believe it's most helpful one!